www/docs/en/dev/cordova/storage/storage.md

```diff
@@ -270,7 +270,7 @@ For more information, see:
 
 - Not supported on iOS.
 - Complex API with nested callbacks.
-- Limited total amount of storage (typically around 5MB).
+- Limited total amount of storage (typically around 5MB) and possible eviction [as described on MDN][MDNIndexedDBLimitsAndEviction].
```

Comment:
5MB is totally wrong. We should update this value, too. IMO we should remove the braces statement and just link to MDN.

Reply:
```suggestion
- Limited total amount of storage and possible eviction [as described on MDN][MDNIndexedDBLimitsAndEviction].
```

@timbru31 would you approve with this change to remove "(typically around 5MB)"?